versions.txt

```diff
@@ -1,3 +1,4 @@
+3.1 - Shooting Galaery Live accurate, Web API, and many QOL fixes
```

Comment:
```suggestion
3.1 - Shooting Gallery Live accurate, Web API, and many QOL fixes
```

Reply:
I swear I know how to spell, I just can't type 😂💀